File: Abundant Number/file.py
def abundant(n):
    sumval=0
    for i in range(1,n):
        if(n%i==0):
            sumval=sumval+i
    return 1 if(sumval>n) else 0 

num=int(input())
x=abundant(num)
print("Yes it's an Abundant Number") if(x==True) else print("No it's not an Abundant Number")

File: Abundant Number/file.c
#include<stdio.h>
#include<stdlib.h>

int abundant(int num){
    int sum=0;
    for(int i=1;i<num;i++){
        if(num%i==0){
            sum+=i;
        }
    }
    return (sum>num)?1:0;
}
int main()
{
    int num;
    scanf("%d",&num);
    int x=abundant(num);
    x==1 ? printf("Yes it's an Abundant Number.") : printf("No it's not an Abundant Number");
}
